client/src/overview/components/addToCart.jsx

```diff
@@ -21,6 +25,28 @@ class AddToCart extends React.Component {
     this.setQuantity = this.setQuantity.bind(this);
     this.closeQuantityDropdown = this.closeQuantityDropdown.bind(this);
     this.setSelectedQuantity = this.setSelectedQuantity.bind(this);
+    this.openSizeDropdown = this.openSizeDropdown.bind(this);
+    this.addItemToCart = this.addItemToCart.bind(this);
+    this.checkCart = this.checkCart.bind(this);
+  }
+
+  checkCart() {
+    axios({
```

Comment:
Oh, this is my first time to see  axios is used in react component. I thought it can only be used in backend(like server). It's a good learning for me!